.github/workflows/ie-tests.yml

```
@@ -0,0 +1,22 @@
+name: IE 11
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build_and_test:
+    if: github.repository == 'ampproject/amphtml'
+    runs-on: windows-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Install Dependencies
+        run: bash ./.github/workflows/install_dependencies.sh
+      - name: Build and Test
+        run: node build-system/pr-check/ie-tests.js
```

It seems odd that you've created this flag in `build-system/pr-check/browser-tests.js` but then still continue to call these files directly. Is there a reason to have both?
```suggestion
        run: node build-system/pr-check/browser-tests.js --browser=edge
```

---

Good observation. The reason this difference exists is that `browser-tests.js` is designed to run on CircleCI in a Docker VM, where it can reuse previously built binaries and run all tests in 3-5 mins. Meanwhile, `ie-tests.js` is a legacy script that runs on GH Actions in a Windows VM, and needs to build the runtime from scratch before running integration tests on IE. It also doesn't run unit / E2E tests.

The plan is to delete `ie-tests.js` later this month once we fully remove all IE 11 code from the repo. This will happen in #35341 (will need to rebase after this PR lands).